server/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "reflextion-server",
+  "version": "0.0.1",
+  "description": "Server-side for reflextion app",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc",
+    "start": "tsc -w & nodemon dist",
+    "lint": "eslint src & prettier src --write",
+    "test": "jest"
+  },
+  "author": "Dan Stetsenko",
+  "license": "ISC",
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged && npm run test --forceExit"
+    }
+  },
+  "lint-staged": {
```

Comment:
husky pre commit hook can be performed without ling staged.
Just use "husky install" and then npx husky add .husky/pre-commit "npm run test && npm run lint"